docs/api-reference/api-reference.md

```diff
@@ -24,864 +24,24 @@ sidebar_label: API endpoints reference
   -->
 
 
-This topic documents all of the API endpoints for each Druid service type.
+This topic is an index to the Apache Druid API documentation.
 
-## Common
+## HTTP APIs
```

Comment:
Let's add brief descriptions to the bullets. Sorry I wasn't clear a about that.

For example
```
* [Automatic compaction](./automatic-compaction-api.md) to optimize segment sizes after ingestion.
```